### PR TITLE
hotfix: bad how-to frontmatter

### PR DESCRIPTION
**Pull request by jorgeorpinel**
ref https://github.com/iterative/dvc.org/pull/1976#issuecomment-735018384

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 'How to Add Dependencies or Outputs to a Stage'
-description: 'It's possible to add files or directories to existing stages, with
-or without executing them.'
+description: "It's possible to add files or directories to existing stages,
+with or without executing them."
```
Comment (jorgeorpinel) [inline content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md:4]: p.s. @rogermparent any way you can think that we could catch these in the PR checks? It wasn't in https://github.com/iterative/dvc.org/pull/1976/checks?check_run_id=1465824496 (right before merge). Thanks!

**Comment by jorgeorpinel** [comment]
Note for self:

- [ ] Do https://github.com/iterative/dvc.org/pull/1976#pullrequestreview-540208751 after this is deployed.

File: content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md (hunk 1 above)
Comment (rogermparent) [reply content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md:4]: This is certainly something I should look into! To be specific, are we talking about checking for lint/format issues on MD frontmatter?

Comment (jorgeorpinel) [reply content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md:4]: @rogermparent the issue above was that I put a single-quote inside the single-quoted string. Seemed to work locally so I missed it, and it passed the PR checks, but it failed to build of course.

**Pull request merged**